helpers/lib/common.js

```diff
@@ -1,10 +1,14 @@
 'use strict';
-const URL = require('url').URL;
+const NodeJsURL = require('url').URL;
 const SafeString = require('handlebars').SafeString;
 
 function isValidURL(val) {
     try {
-        new URL(val);
+        if (NodeJsURL) {
+            new NodeJsURL(val);
+        } else {
+            new URL(val);
```

Comment:
What is the difference between the 2 methods. If vanilla JS method provides the same capabilities then do we need to do this node js check ? can't we simply just switch over to the JS version.

Reply:
You are right. I thought NodeJs supports only package requiring.